src/main/java/io/codelex/polymorphism/practice/exercise4/Staff.java

```diff
@@ -0,0 +1,27 @@
+package io.codelex.polymorphism.practice.exercise4;
+
+public class Staff {
+    static StaffMember[] staffList;
+
+
+    //-----------------------------------------------------------------
+    // Sets up the list of staff members.
+    //-----------------------------------------------------------------
+
```

Comment:
te kods pazudis, tāpēc neiet? 😄 just guessing
un itkā palaidu tavu kodu(vienīgi šeit izmantoju metodi), seems like working for me :smile: iekope screenshot ar error